### ISSUE TITLE
info plugin : nothing displayed when keywords related to plugins

### ISSUE THREAD

**Issue by chtiland** [splitbrain/dokuwiki#2798]
Hello !

All keyword about plugins are not working (nothing displayed)

e.g. : `~~INFO:syntaxplugins~~`

Also, it would be nice to add another keyword like _pluginslist_ to get a list of all installed plugins with version and status (enable or not) to help reporting for some other stuff about Dokuwiki (like other plugins / templates).

Thanks :)

**Comment by phy25** [comment]
Hello, may we get your DokuWiki version? It works: https://www.dokuwiki.org/playground:issue2798

**Comment by chtiland** [comment]
Hello !
DW : Release 2018-04-22b "Greebo"

In fact I use `~~INFO:` several times on the same page... and finally I found that I have a blank page when using `~~INFO:adminplugins~~`.

So I tried to test on another DW site, and `~~INFO:adminplugins~~` works fine... When I try to access directly to Administration (`&do=admin`) I also had a "blank" page. It seems another plugin is responsible. => I Close this issue.
